.github/workflows/release-build.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v3
         with:
-          images: ghcr.io/${{ github.repository_owner }}/keda-cosmosdb-scaler
+          images: ghcr.io/${{ github.repository }}
```

Comment:
This is ok, but if we rename the repo then we will break that - Not sure if we want to have that risk?